DcCore/DcCore/Extensions/UserDefaults+Extensions.swift

```diff
@@ -3,7 +3,26 @@ public extension UserDefaults {
     static var hasExtensionAttemptedToSend = "hasExtensionAttemptedToSend"
     static var hasSavedKeyToKeychain = "hasSavedKeyToKeychain"
     static var upgradedKeychainEntry = "upgradedKeychainEntry_"
+    static var mainAppRunningKey = "mainAppRunning"
+    static var nseFetchingKey = "nseFetching"
+
     static var shared: UserDefaults? {
```

Comment:
idk, maybe it is about that "shared" is not always needed and the computation was considered slow. otoh, it is recalculated on each access.

tbh, i assume there was not much thinking about that, otherwise i would have expected a comment. we can go for what is considered "best practise" at some point